core/arch/arm/include/arm64.h

```diff
@@ -36,6 +36,8 @@
 #define SCTLR_C		BIT32(2)
 #define SCTLR_SA	BIT32(3)
 #define SCTLR_I		BIT32(12)
+#define SCTLR_WXN	BIT32(19)
+#define SCTLR_UWXN	BIT32(20)
```

Comment:
I can't find this bit defined in AArch64 state any where, where did you find it?